tests/integration/update_cluster/minimal_scaleway/kubernetes.tf

```diff
@@ -169,31 +175,40 @@ resource "scaleway_iam_ssh_key" "kubernetes-scw-minimal-k8s-local-be_9e_c3_eb_cb
   public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDKqbVEozfAqng0gx8HTUu69EppcE5SWet6MpwrGShqMVUC4wkoiuVtJDPhMmWmdt7B7Ttc5pvnAZAZaQ6TKMguyBoAyS7qOTLU9/hM803XtSiwQUftOXiJfmsqAXEc8yDyb7UnrF8X7aA3gQJsnQBGJGdp+C88dPHNZenw4PnQc8BNYTCXG9d8F5vJ3xQ5qbiG4HVNoQ2CZh2ht+GedZJ3hl9lMJ24kE/cbMCLKxabMP4ROetECG6PU251jnm84NA8rm0Av/JMmn/c9CFAe0D0D1dGDlHWPsk4mbhGKJ0yU0YliatmPfmgSasismbYzIFf7VPq91ARzRUbavd1fYMBmkMsce0YR/5FdtrpzRhqDzuvwQgQRsoTcttdvp0puFcrtNefMfk8NCbBedIlkzOFxfGiBbe6jde4wqsqEnSrNHwZ2b+Er8z7vjcDPBqYk3gubmMBCrYxg6o1lOS6tTN0kJDUlyKO2AN1ZDr3mpkbhkvZV/N7gLglcClM0X5X7iM= leila@leila-ThinkPad-T14s-Gen-2i"
 }
 
-resource "scaleway_instance_ip" "control-plane-fr-par-1" {
+resource "scaleway_instance_ip" "control-plane-fr-par-1-0" {
+  tags = ["noprefix=kops.k8s.io/cluster=scw-minimal.k8s.local"]
 }
 
-resource "scaleway_instance_ip" "nodes-fr-par-1" {
+resource "scaleway_instance_ip" "nodes-fr-par-1-0" {
+  tags = ["noprefix=kops.k8s.io/cluster=scw-minimal.k8s.local"]
 }
 
-resource "scaleway_instance_server" "control-plane-fr-par-1" {
-  image = "ubuntu_focal"
-  ip_id = scaleway_instance_ip.control-plane-fr-par-1.id
-  name  = "control-plane-fr-par-1"
-  tags  = ["noprefix=kops.k8s.io/cluster=scw-minimal.k8s.local", "noprefix=kops.k8s.io/instance-group=control-plane-fr-par-1", "noprefix=kops.k8s.io/role=ControlPlane"]
-  type  = "DEV1-M"
+resource "scaleway_instance_server" "control-plane-fr-par-1-0" {
+  enable_dynamic_ip = true
+  image             = "ubuntu_focal"
+  ip_id             = scaleway_instance_ip.control-plane-fr-par-1-0.id
+  lifecycle {
+    ignore_changes = [additional_volume_ids]
+  }
+  name                   = "control-plane-fr-par-1-0"
+  replace_on_type_change = false
+  tags                   = ["noprefix=kops.k8s.io/cluster=scw-minimal.k8s.local", "noprefix=kops.k8s.io/instance-group=control-plane-fr-par-1", "noprefix=kops.k8s.io/role=ControlPlane"]
+  type                   = "DEV1-M"
   user_data = {
-    "cloud-init" = filebase64("${path.module}/data/scaleway_instance_server_control-plane-fr-par-1_user_data")
+    "cloud-init" = file("${path.module}/data/scaleway_instance_server_control-plane-fr-par-1-0_user_data")
   }
 }
 
-resource "scaleway_instance_server" "nodes-fr-par-1" {
-  image = "ubuntu_focal"
-  ip_id = scaleway_instance_ip.nodes-fr-par-1.id
-  name  = "nodes-fr-par-1"
-  tags  = ["noprefix=kops.k8s.io/cluster=scw-minimal.k8s.local", "noprefix=kops.k8s.io/instance-group=nodes-fr-par-1"]
-  type  = "DEV1-M"
+resource "scaleway_instance_server" "nodes-fr-par-1-0" {
+  enable_dynamic_ip      = true
+  image                  = "ubuntu_focal"
```

Comment:
Could this be put in a variable to be quickly change for the next LTS when it's out?

Reply:
Same here, the default value for the image is defined [there](https://github.com/kubernetes/kops/blob/9816fd941624f73e5b2a819bce58f443d703042e/upup/pkg/fi/cloudup/populate_instancegroup_spec.go#L60) and updated when possible.